.github/workflows/java_tests.yml

```
@@ -18,38 +18,40 @@
 # To learn more about GitHub Actions in Apache Beam check the CI.md
 
 name: Java Tests
-
 on:
   workflow_dispatch:
     inputs:
       runDataflow:
         description: 'Type "true" if you want to run Dataflow tests (GCP variables must be configured, check CI.md)'
-        default: false
+        default: 'false'
+        required: true
```

```suggestion
```

---

Or:
```suggestion
        required: false
```

If GitHub requires this field